packages/trace-viewer/src/ui/actionList.tsx

```diff
@@ -164,6 +164,13 @@ export function renderTitleForCall(action: ActionTraceEvent): { elements: React.
     title.push(chunk);
 
     const param = formatProtocolParam(action.params, quotedText);
+    if (param === undefined) {
+      elements.push(fullMatch);
+      title.push(fullMatch);
+      currentIndex = match.index + fullMatch.length;
```

Comment:
nit: I'd prefer this line to not be repeated, perhaps `if () {} else if () {} else {}` would be better?